app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -1,17 +1,30 @@
 import React from "react";
 import PropTypes from "prop-types";
+import cx from "classnames";
 
 import Heatmap from "~/components/visualizations/heatmap/Heatmap";
+import { DataTooltip } from "~ui/containers";
+import { getTooltipStyle } from "~/components/utils/tooltip";
 
 import cs from "./amr_heatmap_vis.scss";
 
 const VIEW_LEVEL_ALLELES = "allele";
 const VIEW_LEVEL_GENES = "gene";
 
+const METRICS = [
+  { text: "Coverage", value: "coverage" },
```

Comment:
I would call this "key" instead of "value". Since "value" implies a numerical value for the metric for a specific amr_count.